bcda/models/models_test.go

```diff
@@ -446,239 +440,6 @@ func (s *ModelsTestSuite) TestJobwithKeysCompleted() {
 	s.db.Delete(&j)
 }
 
-func (s *ModelsTestSuite) TestGetEnqueueJobs() {
```

Comment:
Test has been moved to [TestGetQueJobs](https://github.com/CMSgov/bcda-app/pull/568/files#diff-f721bcd60ab3dc5c931f548ae66056f6R398)